strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/io/RepositoryFileSystemProvider.java

```diff
@@ -284,7 +286,7 @@ public void undelete(RepositoryPath path)
         else
         {
             Files.walkFileTree(trashPath.getTarget(),
-                               new MoveDirVisitor(trashPath.getTarget(),
+                               new MoveDirectoryVisitor(trashPath.getTarget(),
                                                   path.getTarget(),
                                                   StandardCopyOption.REPLACE_EXISTING));
         }
```

Comment:
Thanks for renaming the visitor! The name looks better now. Could you just re-indent the method's parameter's above as well? :)